backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/document/model/impl/ElasticsearchIndexCollectorImplMetadata.java

```diff
@@ -28,8 +28,8 @@
 /**
  * @author Yoann Rodiere
  */
-public class ElasticsearchIndexSchemaCollectorImpl<T extends TypeMapping>
-		implements ElasticsearchIndexSchemaElement, IndexSchemaCollector,
+public class ElasticsearchIndexCollectorImplMetadata<T extends TypeMapping>
```

Comment:
Same as below + the `Impl` part is not in the right place.

Reply:
Fixed the Impl part.